src/SamUser/Entity/User.php

```diff
@@ -230,6 +231,44 @@ public function getRoles()
      */
     public function addRole($role)
     {
-        $this->roles[] = $role;
+        $this->roles->add($role);
+    }
+    
+    /**
+     * Remove a role from the user.
+     *
+     * @param Role $role
+     *
+     * @return void
+     */
```

Comment:
I did this as well, but is there any chance you could remove all "return void" docblocks? "return void" is bad :) Was wrong of me in the first place tho ;) not your bad ^^